android-client-sdk/build.gradle

```diff
@@ -30,6 +32,12 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+
+        // Debug build type to be used for local development & unit tests
+        debug {
```

Comment:
wont this break the debugger or stack traces or something?

Reply:
Nope, everything will still work when trying to debug and it will still log full stack traces in the tests as they are using the VERBOSE log level

Reply:
updated with a new build type.. `debug` no longer minifies